phpstan.neon.dist

```diff
@@ -7,9 +7,12 @@ parameters:
 	level: 9
 	paths:
 		# TODO: Move the PHP into an inc/ folder and then use that path. This will allow use of result cache.
```

Comment:
@szepeviktor by the way, are you familiar with this? Currently when I run phpstan in verbose mode, I get:

> Result cache not used because only files were passed as analysed paths.

Is this truly only resolved by passing one or more directories? Seems somewhat of a strange requirement to use caching.

Reply:
> Seems somewhat of a strange requirement to use caching.

PHPStan is an OOP-only too. In normal OOP code you have `src/` with PSR-4-named files.
That is the reason.

Reply:
Actually "strange" goes for the WordPress ecosystem.

Reply:
Strange, yes, but understandably so. The WordPress ecosystem is much older than PSR-4 (this being the 20th year), and backwards-compatibility is a main goal in WordPress.